flaskr/__init__.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, request, render_template
+from src import arbitrage
+
+
+def create_app(test_config=None):
+    # flask server
+    app = Flask(__name__, instance_relative_config=True)
+
+    @app.route('/', methods=("GET", "POST"))
```

Comment:
Why is there still a POST?

you should @app.route('/stock/<ticker>') or something instead of using a query to determine ticker and have the main page just return index.html

Reply:
resolved